pages/about-coq.html

```diff
@@ -63,7 +63,7 @@
 <div class="frameworklabel">Credits</div>
 <div class="frameworkcontent">
 
-    <p>Coq is the result of about 30 years of research. It started in
+    <p>Coq is the result of more than 30 years of research. It started in
     1984 from an implementation of the Calculus of Constructions at
     INRIA-Rocquencourt by Thierry Coquand and Gérard Huet. In 1991,
     Christine Paulin extended it to the Calculus of Inductive
```

Comment:
(below):

"More than a hundred people have contributed"

That sounds low, maybe.

Reply:
Sure.  FWIW, saying "hundreds of people" would be technically correct but a little misleading.